src/test/java/org/gridsuite/modification/server/modifications/StaticVarCompensatorCreationInBusBreakerTest.java

```diff
@@ -56,8 +56,10 @@ protected ModificationInfos buildModification() {
                 .busOrBusbarSectionId("bus1")
                 .connectionName("top")
                 .connectionDirection(ConnectablePosition.Direction.TOP)
-                .maxSusceptance(224.0)
-                .minSusceptance(200.0)
+                .maxSusceptance(null)
```

Comment:
why set null is needed ?

Reply:
the test the case of max and min susceptance is null and max and min Q is not null